docs/gpu.md

```diff
@@ -0,0 +1,153 @@
+| Quadro P4200            | 6.1                |
+| Quadro P3200            | 6.1                |
+| Quadro P5000            | 6.1                |
+| Quadro P4000            | 6.1                |
+| Quadro P3000            | 6.1                |
+| Quadro P2200            | 6.1                |
+| Quadro P2000            | 6.1                |
+| Quadro P1000            | 6.1                |
+| Quadro P620             | 6.1                |
+| Quadro P600             | 6.1                |
+| Quadro P500             | 6.1                |
+| Quadro P520             | 6.1                |
+| Tesla M60               | 5.2                |
+| Tesla M40               | 5.2                |
+| Quadro M6000 24GB       | 5.2                |
+| Quadro M6000            | 5.2                |
+| Quadro M5000            | 5.2                |
+| Quadro M5500M           | 5.2                |
+| Quadro M4000            | 5.2                |
+| Quadro M2200            | 5.2                |
+| Quadro M2000            | 5.2                |
+| Quadro M620             | 5.2                |
+| Quadro K2200            | 5.0                |
+| Quadro K1200            | 5.0                |
+| Quadro K620             | 5.0                |
+| Quadro M1200            | 5.0                |
+| Quadro M520             | 5.0                |
+| Quadro M5000M           | 5.0                |
+| Quadro M4000M           | 5.0                |
+| Quadro M3000M           | 5.0                |
+| Quadro M2000M           | 5.0                |
+| Quadro M1000M           | 5.0                |
+| Quadro K620M            | 5.0                |
+| Quadro M600M            | 5.0                |
+| Quadro M500M            | 5.0                |
+
+## AMD Radeon
+
+Ollama leverages the AMD ROCm library, which does not support all AMD GPUs. In
+some cases you can force the system to try to use a similar LLVM target that is
+close.  For example The Radeon RX 5400 is `gfx1034` (also known as 10.3.4)
+however, ROCm does not currently support this target. The closest support is
+`gfx1030`.  You can use the environment variable `HSA_OVERRIDE_GFX_VERSION` with
+`x.y.z` syntax.  So for example, to force the system to run on the RX 5400, you
+would set `HSA_OVERRIDE_GFX_VERSION="10.3.0"` as an environment variable for the
+server.  If you have an unsupported AMD GPU you can experiment using the list of
+supported types below.
+
+At this time, the known supported GPU types are the following LLVM Targets.
+This table shows some example GPUs that map to these LLVM targets:
+| **LLVM Target** | **An Example GPU** |
+|-----------------|---------------------|
+| gfx900 | Radeon RX Vega 56 |
+| gfx906 | Radeon Instinct MI50 |
+| gfx908 | Radeon Instinct MI100 |
+| gfx90a | Radeon Instinct MI210 |
+| gfx940 | Radeon Instinct MI300 |
+| gfx941 | |
+| gfx942 | |
+| gfx1030 | Radeon PRO V620 |
+| gfx1100 | Radeon PRO W7900 |
+| gfx1101 | Radeon PRO W7700 |
+| gfx1102 | Radeon RX 7600 |
```

Comment:
For 0.1.29, Jeff and Michael built out a more exhaustive list of Radeon cards that should be supported.  We might want to inject that here.